Translator/Translator/Header/Triangle.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+#include "Point.h"
+
+class Triangle {
+public:
+    Point p1;
+    Point p2;
+    Point p3;
+

```

Comment:
make sure to initialize data members as private

Reply:
Ok, data members are made private.